packages/design-systems/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@tailor-platform/design-systems",
-  "version": "0.29.0",
+  "version": "0.29.1",
```

Changes in this PR introduces breaking changes related to types.

```suggestion
  "version": "0.30.0",
```